joss_paper/paper.md

```diff
@@ -24,101 +24,84 @@ bibliography: paper.bib
 ---
 
 # Summary
-TensorFlow.jl is the Julia ([@julia2014]) client library for the TensorFlow deep-learning framework ([@tensorflow2015],[@tensorflow2016]).
-This is accomplished by defining methods which define tensorflow graphs;
-which can then be executed to perform training or inference.
-The methods are defined by overloading functions to operate on a `Tensor` type, which represents a node in the graph.
-In doing so, a lot of standard julia functionality can be made to work.
-Including often allowing code that is unaware of TensorFlow manipulate the compuational graph via _duck-typing_.
-TensorFlow.jl has elegant syntax.
-It allows all the usual infix functions such as `+`, `-`, `.*` etc.
-Further it allows Julia-style indexing (e.g. `x[:, ii + end÷2]), and concentration (e.g. `[A B]`, `[x; y; 1]`).
-It's goal is to be idiomatic for Julia users,
-while still preserving all the power and maturity of the TensorFlow system.
+TensorFlow.jl is a Julia ([@julia2014]) client library for the TensorFlow deep-learning framework ([@tensorflow2015],[@tensorflow2016]).
+It allows users to define TensorFlow graphs using Julia syntax, which are interchangable with the graphs produced by Google's first-party Python TensorFlow client and can be used to perform training or inference on machine-learning models.
 
+Graphs are primarily defined by overloading native Julia functions to operate on a Julia `Tensor` type, which represents a node in a computational graph. This overloading is powered by Julia's powerful multiple-dispatch system, which in turn allows allows the vast majority of Julia's existing array-processing functionality to work as well on the new `Tensor` type as they do on native Julia arrays. User code is often unaware and thereby reusable with respect to whtether its inputs are TensorFlow tensors or native Julia arrays by utilizing _duck-typing_.
 
+TensorFlow.jl has an elegant, idiomatic Julia syntax. It allows all the usual infix operators such as `+`, `-`, `*` etc. It works seamlessly with Julia's broadcast syntax as well, such as the `.*` operator. This `*` can correspond to matrix multiplication while `.*` corresponds to element-wise multiplication, while Python clients need an explicit `matmul` function. It also allows Julia-style indexing (e.g. `x[:, ii + end÷2]`), and concatenation (e.g. `[A B]`, `[x; y; 1]`). Its goal is to be idiomatic for Julia users, while still preserving all the power and maturity of the TensorFlow system. For example, it allows Julia code to operate on TPUs by virtue of using the same TensorFlow graph syntax as Python's TensorFlow client, even though there is no native Julia TPU compiler.
```

Comment:
I just looked this up in the source for python Tensorflow

 - `matmul` is called by the use of infix `@`, which corresponds to our `*`.
 - and `multiply` is called by the use of `*` which corresponds to our `.*`

Can you tweak this?

Reply:
Fixed it